_data/projects/wemake-python-styleguide.yml

```diff
@@ -0,0 +1,12 @@
+name: wemake-python-styleguide
+desc: The strictest and most opinionated python linter ever!
+site: https://github.com/wemake-services/wemake-python-styleguide/blob/master/CONTRIBUTING.md
```

Comment:
@sobolevn Please make this as your repo link i.e "https://github.com/wemake-services/wemake-python-styleguide"